sycl/doc/extensions/supported/sycl_ext_oneapi_bfloat16.asciidoc

```diff
@@ -307,11 +304,11 @@ int main(int argc, char *argv[]) {
 
 == New bfloat16 math functions
```

Comment:
"Example" I thought was not part of the spec. But the "New bfloat16 math functions" section is. So either Example can be moved way down, or we can treat it as an embedded example within the spec. So, OK, I'll indent as you suggest.

Reply:
Either way is fine.  I agree that "Example" not a formal part of the spec, so it would be quite reasonable to move it to the bottom of the document, making it a peer of "Specification".  I actually like that even better.